src/test/java/org/jenkinsci/plugins/workflow/steps/scm/SCMStepTest.java

```diff
@@ -45,118 +45,108 @@
 import static org.junit.Assert.*;
 import org.junit.ClassRule;
 import org.junit.Rule;
-import org.junit.runners.model.Statement;
 import org.jvnet.hudson.test.BuildWatcher;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.RestartableJenkinsRule;
 
 public class SCMStepTest {
 
     @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();
-    @Rule public RestartableJenkinsRule r = new RestartableJenkinsRule();
+    @Rule public RestartableJenkinsRule rr = new RestartableJenkinsRule();
```

Comment:
🐜  Changed it to make it easier to see where you needed to rewrite the usage and then didn't revert that, or what?  
Harmless, of course.